docs/user_guide/resource_quota_mgmt.md

```diff
@@ -121,80 +142,104 @@ partitions:
 
-Continue to create namespaces like before, do not create namespace quota anymore.
-Instead, set the following annotation in the namespace object:
+### Namespace quota
+Namespaces in Kubernetes contain the quota information. 
+If a quota is set on a namespace Kubernetes will automatically enforce the quota.
+In the case that YuniKorn is used for quota enforcement no quota must be set on the namespace.
 
+To allow specifying a quota on the namespace the following annotations should be set in the namespace object:
 ```yaml
 yunikorn.apache.org/namespace.max.cpu: "64"
 yunikorn.apache.org/namespace.max.memory: "100Gi"
 ```
+YuniKorn will parse these annotations and set the maximum capacity of the queue mapped to this namespace.
+The values specified follow the standard Kubernetes formatting and unit specification.
+Currently, we only support mapping memory and cpu not other resource types.
 
-YuniKorn will parse the annotation and set the max capacity of the dynamical queue
-that mapped to this namespace to 64 CPU and 100GB memory.
+The example above will limit the queue mapped to the annotated namespace to 64 CPUs and 100GB memory.
 
-### Run workloads
+### Run a workload
 
-Jobs continue to be submitted to namespaces, based on the `Placementrule` used
-in the configuration. YuniKorn will automatically run the job and all its pods in
-the corresponding queue. For example, if a job is submitted to namespace `development`,
-then you will see the job is running in `root.development` queue.
+Applications, and the pods that are part of the application, can be submitted without specific labels. 
+YuniKorn will automatically add the required tags.
+The configured placement rule will create the queue, if required, and add the application to the queue.
+ 
+For example, if an application is submitted to namespace `development`, then the application will run in the `root.development` queue.
 
-## Option 3) Hierarchy queues with dynamical leaves that mapped to namespaces
+## Parent queue mapping
 
 ### Goal
-Though the tag placement rule using the `namespace` tag is capable of putting applications based on the name of the namespace, sometimes more dynamic placement is required.
+Though the tag placement rule using the `namespace` tag is capable of placing an application in a queue this might not be enough in all setups.
+In some cases, multi tenancy for example, namespaces need to be grouped together.
+Administrators could annotate namespaces which allows dynamic placement of applications based on multiple annotations if placement rules were setup.
+YuniKorn cannot and does not just add all annotations from a namespace to an application.
 
-Users can annotate namespaces which allows dynamic placement of applications based on the annotation value if proper placement rules are present.
+To help support this grouping case a parent queue can be tagged on a namespace.   
 
 ### Configuration
```

Comment:
Below section looks very complex to understand.

Reply:
Streamlined and moved things around. The diagram should also help.